### DISCUSSION TITLE
Application not reviewed but its over the deadline

### DISCUSSION THREAD

**Discussion by RScriptz** [community/community#125643] [GitHub Education] [answered]
### Hi everyone,

Hi, I applied for the 3rd time, after i fixed my previous applications, and they said I was going to get a response by may 24, but I still did not get any response, is it normal and i should wait a bit more?

**Comment by 2998508** [comment]
Me too

**Reply by RScriptz** [reply]
Ill update this post once it gets reviewed (and approved hopefully)

**Comment by KlausJackson** [comment]
Same, I applied for the second time after weeks of waiting for my first application to be processed and accepted. I think they forgot lol.

At least you got a response email, I don't have any email saying when I will hear back from them.

**Reply by RScriptz** [reply]
Dang, first 2 times they responded almost instantly, then a few days before the deadline, now they didnt, do you know any way to contact them or something?

**Reply by KlausJackson** [reply]
You can try to create a ticket for support, that's the only solution left now but I haven't tried this yet.
https://support.github.com/tickets/personal/0

Tell me when you got accepted.

**Reply by RScriptz** [reply]
Thanks

**Comment by y0geshx** [comment]
Before Applying did you follow  these points : 

- Check that you are using a [supported browser](https://docs.github.com/en/get-started/using-github/supported-browsers), and that location - services are not blocked by your browser or platform.
- Complete your GitHub account [billing information](https://github.com/settings/billing/payment_information) with your full legal name as it appears on your academic affiliation documentation. (You do not have to add a payment method.)
[Verify](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/adding-an-email-address-to-your-github-account) your academic email address on your GitHub account, if your school provides one.
- Secure your GitHub account with [two-factor authentication](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa). (We recommend using the [GitHub Mobile](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-github-mobile) app.)
[Personalize](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) your public GitHub Profile with your photo, your name, your pronouns, and more.
- Generate a README for your public GitHub Profile, [customize it with your interests](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme), and publish it for the world to see.

I also  apply for Student Developer Pack i got confirmation within a week :)
Proof :

![2024-05-26_21-02](https://github.com/community/community/assets/92950980/d0a41896-9336-4b8b-90e3-9a104b8ed7bc)

**Reply by RScriptz** [reply]
I didnt do the last one and some of thr 2nd last one, everything else i did except that my school doesnt provide an email, even if there was something wrong with the application they usually (atleast till now) deny it and tell me whats wrong/what needs fixing, that happened twice already.

**Reply by RScriptz** [reply]
Still not approved/denied to this day, didnt make a support ticket since i cant make any related to my problem atm.

**Reply by KlausJackson** [reply]
Don't worry @RScriptz , I did all that months ago but I got no response back lol.

**Reply by RScriptz** [reply]
Dang

**Reply by KlausJackson** [reply, marked as answer]
Nevermind dude... I just got accepted 🤣 You'll get there eventually! Just keep waiting I guess...

![Screenshot_2024-05-28-12-47-00-971_com.android.chrome-edit.jpg](https://github.com/community/community/assets/140784820/1f68fb17-1472-488b-bec0-678f8746a6cf)

**Reply by RScriptz** [reply]
Dam congrats im still waiting

**Reply by RScriptz** [reply]
Ok so i got declined again, atleast i got the reaponse, they said the billing information name doesnt match the school id name even tho its the same name, i resent it the same as before after checking once again.

**Reply by RScriptz** [reply]
Lmao i made a new application and it got accepted the same minute i got it, even tho they said it can take up to 13 days!

**Reply by KlausJackson** [reply]
That's what I did, I made a new one. I wonder what happened to the old application lol.

**Reply by RScriptz** [reply]
Probably forgot about it, no idea tbf
